captum/attr/_core/lime.py

```diff
@@ -0,0 +1,1007 @@
+                    be a single tensor of shape 1 x num_interp_features,
+                    corresponding to the representation of the
+                    sample to train the interpretable model.
+
+                    Note that kwargs include all kwargs passed to the attribute
+                    method.
+            sample_interpretable_space (bool, optional): Indicates whether
+                    sampling_func returns a sample in the interpretable space
+                    (tensor of shape 1 x num_interp_features) or a sample
+                    in the original space, matching the format of the original
+                    input. Once sampled, inputs can be converted to / from
+                    the interpretable representation with either
+                    to_interp_rep_transform or from_interp_rep_transform.
+            from_interp_rep_transform (callable): Function which takes a
+                    single sample's interpretable representation (tensor
+                    of shape 1 x num_interp_features) and returns
+                    the corresponding input in the original input space
+                    (mathing shapes of original input to attribute).
+
+                    This argument is necessary if sample_interpretable_space
+                    is True, otherwise None can be provided for this argument.
+
+                    The expected signature of this callable is:
+
+                    from_interp_rep_transform(
+                        curr_sample: Tensor [2D 1 x num_interp_features]
+                        original_input: Tensor or Tuple of Tensors,
+                        **kwargs: Any
+                    ) -> Tensor or tuple of Tensors
+
+                    Returned sampled input should match the type of original_input
+                    and corresponding tensor shapes.
+
+                    Note that kwargs include all kwargs passed to the attribute
+                    method.
+
+            to_interp_rep_transform (callable): Function which takes a
+                    sample in the original input space and converts to
+                    its interpretable representation (tensor
+                    of shape 1 x num_interp_features) and returns
+                    the corresponding input in the original input space.
+
+                    This argument is necessary if sample_interpretable_space
+                    is False, otherwise None can be provided for this argument.
+
+                    The expected signature of this callable is:
+
+                    to_interp_rep_transform(
+                        curr_sample: Tensor or Tuple of Tensors,
+                        original_input: Tensor or Tuple of Tensors,
+                        **kwargs: Any
+                    ) -> Tensor [2D 1 x num_interp_features]
+
+                    curr_sample will match the type of original_input
+                    and corresponding tensor shapes.
```

Comment:
This is primarily for consistency with from_interp_rep_transform, but it could be useful if the interpretable representation is related to the original input, e.g. the default interpretable representation mixing baseline and original input.